baseband/gsb/base.py

```diff
@@ -176,40 +176,42 @@ def __repr__(self):
 class GSBStreamReader(GSBStreamBase, VLBIStreamReaderBase):
     """GSB format reader.
 
-    Requires both a timestamp and one or more corresponding raw data files.
+    Allows access to GSB files as a continuous series of samples.  Requires
+    both a timestamp and one or more corresponding raw data files.
 
     Parameters
     ----------
     fh_ts : `~baseband.gsb.base.GSBTimeStampIO` instance
-        Header file handle.
-    fh_raw : `~io.BufferedReader` instance, or nested tuple of instances
-        Raw binary data file handle(s).  A single file is needed for rawdump,
+        Header filehandle.
+    fh_raw : filehandle, or nested tuple of filehandles
+        Raw binary data filehandle(s).  A single file is needed for rawdump,
         and a tuple for phased.  For a nested tuple, the outer tuple determines
         the number of polarizations, and the inner tuple(s) the number of
         streams per polarization.  E.g., ((polL1, polL2), (polR1, polR2)) for
         two streams per polarization.  A single tuple is interpreted as
         streams of a single polarization.
     sample_rate : `~astropy.units.Quantity`, optional
-        Number of complete samples per second (ie. the rate at which each
+        Number of complete samples per second (i.e. the rate at which each
         channel of each polarization is sampled).  If not given, will be
         inferred assuming the framerate is exactly 0.25165824 s.
-    nchan : int, optional
-        Number of channels. Default is `None`, which sets it to 1 for rawdump,
-        512 for phased.
-    bps : int, optional
-        Bits per elementary sample (e.g., the real or imaginary part of each
-        complex data sample).  Default is `None`, which sets it to 4 for
-        rawdump, 8 for phased.
-    complex_data : bool, optional
-        Default is `None`, which sets `False` for rawdump, `True` for phased.
     samples_per_frame : int, optional
         Number of complete samples per frame.  Can give ``payloadsize``
         instead.
     payloadsize : int, optional
         Number of bytes per payload, divided by the number of raw files.
-        If both ``samples_per_frame`` and ``payloadsize`` are ``None``,
+        If both ``samples_per_frame`` and ``payloadsize`` are `None`,
         ``payloadsize`` is set to 2**22 (4 MB) for rawdump, and 2**23 (8 MB)
         divided by the number of streams per polarization for phased.
+    nchan : int, optional
+        Number of channels. Default: `None`, which sets it to 1 for rawdump,
```

Comment:
In general, I've written descriptions for arguments even if they seem self-explanatory (better to seem pedantic than confusing).  It also sort of ties in to the glossary, though I don't think explicitly linking the docstrings to the blbliography (if even possible) is a good idea.

Reply:
I think I was not clear here: I meant to remove "`None`, which sets it to`, so that one ends up with `Number of channels. Default: 1 for ...` - I feel this is closer to the style of the other descriptions, and I don't feel there is a sense of noting that the default is indicated by `None` (i.e., I wouldn't want people to pass that in).